docs/details/commands/pmm-admin.md

```diff
@@ -325,10 +325,28 @@ When you remove a service, collected data remains on PMM Server for the specifie
 
 ##### Advanced Options
 
-###### Enable all collectors
-
 PMM starts the MongoDB exporter by default only with `diagnosticdata` and `replicasetstatus` collectors enabled.
 
+    FLAGS:
+
+    `--enable-all-collectors`
+    :  Enable all collectors
+
+    `--disable-collectors`
+    :  Comma-separated list of collector names to exclude from exporter
+
+    `--max-collections-limit=-1`
+    :  Disable collstats & indexstats if there are more than <n> collections. 0: No limit, -1: Let PMM automatically set this value.
```

Comment:
1. Replace **&** with **and**
2. Instead of **Let PMM automatically set this value.** could we write **PMM automatically sets this value.**